Tests/EngineKitTests/DecimalTests.swift

```diff
@@ -616,45 +590,6 @@ extension DecimalTests {
 			"1234123.4",
 			"123456.34",
 			"12345.234",
-			"1234.1234",
```

Comment:
why remove so many strings literals to test?

Reply:
These are just used for testing round trip coding now, which uses the built-in machine readable string representation. So the test can be pretty limited, we don't need to look at rounding or anything like that. Really it's enough to check integers, leading zeros, trailing zeros, negatives and 0. I made the test very exhaustive earlier because we encoded as BigDecimal and the decoded as RETDecimal, that made sense to test more exhaustively.